src/main/java/com/spotify/reaper/service/SegmentRunner.java

```diff
@@ -327,15 +327,15 @@ boolean canRepair(RepairSegment segment, String keyspace, JmxProxy coordinator,
                 + " Run id '{}'", hostName, segment.getRunId());
       JmxProxy hostProxy = null;
       try{
-        Optional<HostMetrics> hostMetrics;
+        Optional<HostMetrics> hostMetrics = Optional.absent();
         try{
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Extract this nested try block into a separate method. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1141)